cmp/example_test.go

```diff
@@ -147,6 +147,7 @@ func ExampleOption_equalEmpty() {
 	// false
 }
 
+
```

Comment:
Would you like a PR that adds a `gofmt` check to Travis CI, so this type of issue can be caught automatically by CI in all future contributions?